sdk/storage/azure-storage-blob/azure/storage/blob/__init__.py

```diff
@@ -183,7 +184,7 @@ def download_blob_from_url(
     overwrite = kwargs.pop('overwrite', False)
     with BlobClient.from_blob_url(blob_url, credential=credential) as client:
         if hasattr(output, 'write'):
-            _download_to_stream(client, output, **kwargs)
+            _download_to_stream(client, output, **kwargs)  # type: ignore [arg-type]
```

Comment:
azure\storage\blob\__init__.py:187: error: Argument 2 to "_download_to_stream" has incompatible type "Union[str, IO[Any]]"; expected "IO[Any]"  [arg-type]

Reply:
Cast output to `IO[bytes]`